### PR TITLE
Add API Library list to /apidocs

### LINKED ISSUE #1315: Add List of API Clients to site
There are some TBA API clients floating around there, we should compile a list and make a section of the site, so it's easier to develop with. Here are some that I know of that look to be in decent shape

 - https://github.com/phil-lopreiato/tba-apiv2-java
 - https://github.com/chacha/thebluealliance-ruby
 - https://github.com/chacha/thebluealliance-javascript
 - https://github.com/Alexanders101/The-Blue-Alliance-Python-API
 - https://github.com/xNovax/TheBlueAlliance_API

### PR DESCRIPTION
**Pull request by chacha**
Created a first run at the API Library list. I don’t have a running copy of TBA yet, so I’m running a little blind. However, this should be fairly trivial. Feel free to clean up anything I’m missing.

Closes #1315

### CODE REVIEW

**Comment by chacha** [comment]
We might want to look at some of the repositories that are on the older side. At some point it might not be helpful for them to be included as users could have problems with them.

File: templates/apidocs.html [hunk 1]
```diff
@@ -1583,6 +1586,60 @@ <h3 class="panel-title">Example GET Request:</h3>
         </div>
       </div>
 
+      <h2 id="api-libraries">API Libraries</h3>
+      <p>
+        These libraries have been created by our community to help you use our
+        API in your next project.
+      </p>
+      <table class=table>
```
Comment (chacha) [inline templates/apidocs.html]: Copy and pasted this. Probably should put some quotation marks around that class.

File: templates/apidocs.html [hunk 2]
```diff
@@ -1583,6 +1586,60 @@ <h3 class="panel-title">Example GET Request:</h3>
...
+          <tr>
+            <td>Java</td>
+            <td><a href="https://github.com/FRC2834/java-blue-alliance-api">java-blue-alliance-api</a></td>
+          </tr>
+          <tr>
+            <td>Java</td>
+            <td><a href="https://github.com/tonypeng/TheBlueAllianceJavaAPI">TheBlueAllianceJavaAPI</td>
+          </tr>
+          <tr>
+            <td>Javascript</td>
```
Comment (mc10) [inline templates/apidocs.html]: Minor nit: This should be "JavaScript" (with a capital S).

**Comment by gregmarra** [comment]
This looks cool to me. I'd love if we could also include the library author's name (or github handle) in this table, to give people a shout out for their support!

**Comment by synth3tk** [comment]
Or the team, if it's from a team's org.

**Comment by gregmarra** [comment]
Going to merge this, we can fix forward with improvements. Thanks @chacha !

**Pull request merged**